flow-report/jest.config.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+module.exports = {
+  testEnvironment: 'jsdom',
+  preset: 'ts-jest',
```

Comment:
does `ts-jest` have advantages over using the plain old babel pipeline jest makes us install?

Reply:
Well I haven't needed to install the babel pipeline yet.

Also babel doesn't type check when tests are run https://babeljs.io/docs/en/babel-plugin-transform-typescript#caveats.

Reply:
> Well I haven't needed to install the babel pipeline yet.

it's in jest's deps. We [turn it off](https://github.com/GoogleChrome/lighthouse/blob/c23fff1c50d608ab0796725fc9747f6c027afa4b/jest.config.js#L36) in our root `jest.config.js`. Presumably just need to set it back to its [default value](https://jestjs.io/docs/next/configuration#transform-objectstring-pathtotransformer--pathtotransformer-object) `{"\\.[jt]sx?$": "babel-jest"}`.

> Also babel doesn't type check when tests are run https://babeljs.io/docs/en/babel-plugin-transform-typescript#caveats.

is that useful vs adding to the other type checks?

Reply:
Ah ok, If we don't need to install anything and can do type checking elsewhere then babel would be better

Reply:
we might have to turn it on for all code soon anyways, because `jest.mock` currently doesn't work in es modules, but the babel transform will inject it